.github/actions/prepare/action.yml

```diff
@@ -13,7 +13,7 @@ runs:
         PNPM_VERSION=$(jq -r '.engines.pnpm' < package.json | sed -r 's/[>=]+//g')
         echo "value=${PNPM_VERSION}" >> "$GITHUB_OUTPUT"
 
-    - uses: pnpm/action-setup@v2.2.4
+    - uses: pnpm/action-setup@v4.0.0
       with:
         version: ${{ steps.get-pnpm-version.outputs.value }}
 
```

Comment:
Might as well update `actions/setup-node` (line below this one) as well.

Latest is `v4.1.0`: https://github.com/actions/setup-node

Reply:
Done!
I though I had done this before and it failed, but actually it's totally fine...